src/effect.cpp

```diff
@@ -1288,3 +1292,49 @@ void effect::deserialize( JsonIn &jsin )
     start_time = calendar::time_of_cataclysm;
     jo.read( "start_turn", start_time );
 }
+
+std::string texitify_base_healing_power( const int power )
+{
+    std::string ret;
+    if( power == 1 ) {
+        ret = string_format( _( "very poor" ) );
+    } else if( power == 2 ) {
+        ret = string_format( _( "poor" ) );
+    } else if( power == 3 ) {
+        ret = string_format( _( "decent" ) );
+    } else if( power == 4 ) {
+        ret = string_format( _( "good" ) );
+    } else if( power >= 5 ) {
+        ret = string_format( _( "great" ) );
+    }
+    if( power < 1 ) {
+        debugmsg( "Tried to convert zero or negative value." );
+    }
+    return ret;
+}
+
+std::string texitify_healing_power( const int power )
+{
+    std::string ret;
+    if( power >= 1 && power <= 2 ) {
+        ret = string_format( _( "poor" ) );
```

Comment:
Why all the unnecessary calls to `string_format`? And have you considered just returning the string from here instead of assigning it and returning it later?